epinorm/norm.py

```diff
@@ -122,6 +129,86 @@ def get_feature_ids(self):
             .dropna()
             .unique()
         )
+    
+    def _admin_levels_table(self):
+        """
+        This method retrives the information in the "countries.csv" and "administrative_units.tsv".
+        It returns a dictionnary that maps a country name (from countries.csv) to a list of its administrative levels.
+        administrative levels are encoded as dictionaries with properties:
+        {
+            name,
+            admin_level,
+            osmId
+        }
+
+        it also tranliterates information from the administrative_units.tsv file.
+        """
+
+        # * create a mapping from country name to its two letter code
+        country_to_code = {}
+        df = pd.read_csv(COUNTRIES_DATA)
+        for _, row in df.iterrows():
+            country_to_code[row["name"]] = row["alpha_2"]
+
+        # add the exceptions
+        country_to_code.update(COUNTRIES_EXCEPTIONS) 
+
+
+        # * now map country code to its admin levels
+        code_to_admin = {}
+        df = pd.read_csv(ADMIN_LEVELS_DATA, sep="\t")
+        for _, row in df.iterrows():
+
+                if row["iso3166_1_code"] not in code_to_admin:
+                    code_to_admin[row["iso3166_1_code"]] = []
+
+                # a row sometimes doesn't contain endonym
+                if type(row["endonym"]) is not float:
+                    entry = {"name": row["endonym"], "admin_level":row["admin_level"], "osmId": row["osm_id"]}
+                    code_to_admin[row["iso3166_1_code"]].append(entry)
+
+                # try to tranlisterate the endonym
+                try:
+                    transliteration = translit(row["endonym"], reversed=True)
+
+                    # if the original language was russian, then it might have used the character ь which
+                    # doesn't get transliterate properly (it gets into an apostrophe)
+                    if "ь" in row["endonym"] and "'" in transliteration:
+                        transliteration = transliteration.replace("'", "")
+
+                    entry = {"name":transliteration, "admin_level":row["admin_level"], "osmId":row["osm_id"]}
+                    code_to_admin[row["iso3166_1_code"]].append(entry)
+                except:
```

Comment:
Usually we want to `except` a specific error type(s) that we expect, but not generically catch all exceptions.